projects/plugins/jetpack/modules/publicize.php

```diff
@@ -111,10 +122,11 @@ function () {
 	function publicize_init() {
 		global $publicize;
 
-		if ( ! class_exists( 'Publicize' ) ) {
+		$in_jetpack = ( class_exists( 'Jetpack' ) && method_exists( 'Jetpack', 'enable_module_configurable' ) ) ? true : false;
```

Comment:
Aren't we always in Jetpack at this point? Or is this more to check if the `Config` changes are available?

Reply:
My bad, this has been removed and as you said, we are always in Jetpack at this point.